nestjs/src/courses/course-tasks/dto/course-task.dto.ts

```diff
@@ -61,6 +62,9 @@ export class CourseTaskDto {
   @ApiProperty()
   studentEndDate: string;
 
+  @ApiProperty({ nullable: true, type: String })
+  studentRegistrationStartDate?: string | null | Date;
```

Comment:
i think it should be `string | null`. it's better always convert to string in dto